crates/hyperqueue/src/common/serverdir.rs

```diff
@@ -144,6 +144,10 @@ pub struct AccessRecord {
     /// Version of HQ
     version: String,
 
+    /// UID of the server instance
+    #[serde(default)]
```

Comment:
Do we need default? We don't guarantee backwards compatibility with older HQ versions for the access record, right?

Reply:
It ends later on an invalid version. I wanted to avoid some ugly message that access.json cannot be parsed while we already have a reasonable error message